tests/models/test_decoders.py

```diff
@@ -99,6 +100,16 @@
 if isinstance(common_max_new_tokens, str):
     common_max_new_tokens = [int(mnt) for mnt in common_max_new_tokens.split(",")]
 
+# pass metrics to skip as a comma separated list (ce, mean_diff)
```

Comment:
does the comma have a space after it or would that break? if so, fix the example